java/gast_to_code/gast_to_code_java.py

```diff
@@ -0,0 +1,69 @@
+import shared.gast_to_code.gast_to_code_router as router
+from shared.gast_to_code.abstract_gast_to_code_converter import AbstractGastToCodeConverter
+
+class JavaGastToCodeConverter(AbstractGastToCodeConverter):
+    def handle_bool(gast):
+        if gast["value"] == 1:
+            return "true"
+        else:
+            return "false"
+
+    def handle_if(gast, lvl=0):
+        pass
+
+    def handle_none(gast):
+        pass
+
+    def handle_while(gast, lvl=0):
+        pass
+
+    def handle_for_range(gast, lvl=0):
+        pass
+
+    def handle_for_of(gast, lvl=0):
+        pass
+
+    def handle_log_statement(gast):
+        return "System.out.println"
+
+    def handle_var_assign(gast):
+        pass
+
+    def handle_aug_assign(gast):
+        pass
+
+    def handle_func_call(gast):
+        return router.gast_to_code(gast["value"], "java") + "(" + router.gast_to_code(gast["args"], "java") + ")"
```

Comment:
Consider making this a constant at the top of the file since it is used in multiple places.